src/utils/stakingClient.ts

```diff
@@ -65,5 +65,7 @@ export const getMakerSupportEmail = (makerId?: number): string => {
       return 'support@syncrob.it'
     case 6:
       return 'support@bobcatminer.com'
+    case 7:
```

Comment:
We will need to wait to confirm that maker ID 7 will be correct.

Reply:
7 is correct.